### PR TITLE
feat: Authentication page

### PR DESCRIPTION
**Pull request by shivangisharma01**
Authentication Page
-SignUp Page
-Login Page
-SignIn Page

image
image

### CODE REVIEW

**Review by adithyark17 — APPROVED** [review]

**Comment by adithyark17** [comment]
Looks good. Please fix the alignment

File: Page/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+.login-heading {
+    font-size: 2rem;
+    font-weight: 800;
+}
+
+.login-input {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (nishasen) [inline Page/Login/login.css]: use util class here

File: Page/Login/login.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+    <title>Scentcy</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-container-box">
+            <div class="nav-logo"><a href="/../index.html">Scentcy <i class="fa-solid fa-spray-can-sparkles"></i></a></div>
+            <input type="search" placeholder="   Search for items here" class="nav-input">
```
Comment (nishasen) [inline Page/Login/login.html]: Remove the extra space in placeholder

File: Page/Logout/logout.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <main>
+            <div class="login-container logout">
+                <div class="login-heading">You have been successfully</div>
+                <div class="login-heading">Logged Out</div>
+
+
+         
+            </div>
+        </main>
+
```
Comment (nishasen) [inline Page/Logout/logout.html]: remove the extra space

File: Page/Signup/signup.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                <div class="login-heading">SignUp</div>
+                <div class="login-input">
+                    <div class="input-group-box ">
+                        <label class="input-label">NAME <span class="mandatory"> *</span> </label>
+
+                        <input type="text" class="input-simple" placeholder="Enter userName here">
+
+
+                    </div>
+
```
Comment (nishasen) [inline Page/Signup/signup.html]: remove the extra space

File: css/navbar.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
+.nav-container-box {
+    background-color: var(--nav-background);
+    width: 100%;
+    display: flex;
+    align-items: center;
+   padding: 1rem;
+
```
Comment (nishasen) [inline css/navbar.css]: remove the extra space

**Review by nishasen — APPROVED** [review]
The code looks clean, some minor changes needed to be done. Please look into the comments.

File: Page/Login/login.css (hunk 1 above)
Comment (shivangisharma01) [reply Page/Login/login.css]: done

File: Page/Logout/logout.html (hunk 3 above)
Comment (shivangisharma01) [reply Page/Logout/logout.html]: done

File: Page/Signup/signup.html (hunk 4 above)
Comment (shivangisharma01) [reply Page/Signup/signup.html]: done

File: css/navbar.css (hunk 5 above)
Comment (shivangisharma01) [reply css/navbar.css]: done

File: Page/Login/login.html (hunk 2 above)
Comment (shivangisharma01) [reply Page/Login/login.html]: done

**Pull request merged**